lib/MatrixGitterBridge.js

```diff
@@ -510,6 +510,22 @@ MatrixGitterBridge.prototype.actionUnlink = function(matrixId, gitterName) {
     });
 };
 
+MatrixGitterBridge.prototype.getLinkByMatrixId = function(matrixId) {
```

Comment:
This function is looking in the data store directly; it should probably work by querying running in-memory state.

I'd name it something like `findRoomByMatrixId`, taking a matrix ID and returning a BridgedRoom instance directly. Also, don't add it at the end of the file with the action methods, put it higher up - maybe at line 260 right after `getOrCreateBridgedRoom`.